robottelo/utils/virtwho.py

```diff
@@ -306,28 +328,70 @@ def deploy_validation(hypervisor_type):
     """
     status = get_virtwho_status()
     if status != 'running':
-        raise VirtWhoError("Failed to start virt-who service")
+        raise VirtWhoError(f"Failed to start virt-who service. Status: {status}")
     hypervisor_name, guest_name = _get_hypervisor_mapping(hypervisor_type)
     for host in Host.list({'search': hypervisor_name}):
         Host.delete({'id': host['id']})
     restart_virtwho_service()
     return hypervisor_name, guest_name
 
 
-def deploy_configure_by_command(command, hypervisor_type, debug=False, org='Default_Organization'):
+def get_virt_who_ak(org):
+    """Create a virt-who activation key for the given organization.
+    :param str org: The label of the organization for which the activation key is created.
+    :raises: Exception: If the organization, lifecycle environment, or content view cannot be found.
+    :return: The name of the newly created activation key.
+    :rtype: str
```

Comment:
```suggestion
    :return str: The name of the newly created activation key.
```
Non-blocking, maybe you could designate the return type like this.